android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -160,19 +159,12 @@ class GestureHandlerOrchestrator(
       activationIndex = this@GestureHandlerOrchestrator.activationIndex++
     }
 
-    // Cancel all handlers that are required to be cancel upon current handler's activation
-    for (otherHandler in gestureHandlers) {
+    for (otherHandler in gestureHandlers.reversed()) {
```

Comment:
```suggestion
    for (otherHandler in gestureHandlers.asReversed()) {
```

I don't think the list is being mutated here, so we shouldn't need a copy.